app/helpers/static_pages_helper.rb

```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```

Comment:
remove it because not use

Reply:
if i remove this, i cannot open my rails